sql/core/src/main/scala/org/apache/spark/sql/execution/joins/BroadcastNestedLoopJoinExec.scala

```diff
@@ -43,6 +46,11 @@ case class BroadcastNestedLoopJoinExec(
     case BuildLeft => (right, left)
   }
 
+  override def simpleStringWithNodeId(): String = {
```

Comment:
do we need to override it?

Reply:
It's to show the `BuildSide` of `BroadcastNestedLoopJoinExec`, I think we missed it here. We don't need to move`BuildSide` to `BaseJoinExec`, right?

Reply:
Yea we don't need